builtin/mainmenu/tab_settings.lua

```diff
@@ -346,6 +342,9 @@ local function formspec(tabview, name, tabdata)
 			"checkbox[3.85,4.3;cb_vrange_auto_adjust;" .. fgettext("Auto-adjust") .. ";" ..
 				dump(core.setting_getbool("viewing_range_auto")) .. "]"
 	end
+	if get_vrange_min > get_vrange_max then
+		core.setting_set("viewing_range_nodes_min", get_vrange_max)
```

Comment:
1. This is always being rendered within the function `formspec()`.
2. This checking deals only with the raw settings, thus that doesn't matter which string the formspec currently has (i.e. it behaves the same than in the if statement).